package.json

```diff
@@ -7,7 +7,7 @@
     "prettier": "prettier --config .prettierrc --write \"./**/*.{css,html,json,md,yml}\"",
     "lint": "eslint --config .eslintrc \"./**/*.js\"",
     "jest": "jest",
-    "test": "cypress install && concurrently --kill-others \"node_modules/servor/servor.js --no-browser --no-reload\" \"wait-on http-get://localhost:8080 && npx cypress run --browser chrome --env env=local\" --success first "
+    "test": "jest"
```

Comment:
Is there much need for having `jest` and `test` scripts (that do the same thing)?

Reply:
IMO the test command will be what we call in CI, we will add cypress tests back in under the test command in the future when we have the UI in a better place.

Reply:
Sounds good!